### PR TITLE
WIP: add percolator

### PR DESCRIPTION
**Pull request by lecorguille**
- [X] I have read the guidelines above.
- [X] This PR adds a new recipe.
- [ ] This PR updates an existing recipe.
- [ ] This PR does something else (explain below).

Only percolator seems ready.
I'm working on converters

Part of https://github.com/galaxyproteomics/tools-galaxyp/issues/57

### CODE REVIEW

**Comment by lecorguille** [comment]
I need xsd to go further : https://github.com/bioconda/bioconda-recipes/pull/2439

**Comment by lecorguille** [comment]
@bioconda/core can you point me the problem and give me clue? 
https://travis-ci.org/bioconda/bioconda-recipes/jobs/163649811#L833

File: recipes/percolator/build.sh [hunk 1]
```diff
@@ -3,6 +3,6 @@ make
 make install
 
 cd src/converters
-cmake . -DSERIALIZE="Boost" -DCMAKE_INSTALL_PREFIX=$PREFIX -DBOOST_INCLUDEDIR=$PREFIX/include/boost/ -DBOOST_LIBRARYDIR=$PREFIX/lib -DBOOST_ROOT=$PREFIX -DCMAKE_PREFIX_PATH=$PREFIX/lib
+cmake . -DSERIALIZE="Boost" -DCMAKE_INSTALL_PREFIX=$PREFIX -DBOOST_INCLUDEDIR=$PREFIX/include/boost/ -DBOOST_LIBRARYDIR=$PREFIX/lib -DBOOST_ROOT=$PREFIX -DSQLITE3_INCLUDE_DIR=$PREFIX/include/sqlite3 -DCMAKE_PREFIX_PATH=$PREFIX/lib
```
Comment (bgruening) [inline recipes/percolator/build.sh]: maybe without the leading sqlite?

Comment (lecorguille) [inline recipes/percolator/build.sh]: I didn't see your review, sorry.
But I was arriving at the same conclusion!

**Pull request closed without merging**